sdk/storage/azure-storage-blob/tests/perfstress_tests/_test_base.py

```diff
@@ -32,8 +33,28 @@ def __init__(self, arguments):
         # self._client_kwargs['api_version'] = '2019-02-02'  # Used only for comparison with T1 legacy tests
 
         if not _ServiceTest.service_client or self.args.no_client_share:
-            _ServiceTest.service_client = SyncBlobServiceClient.from_connection_string(conn_str=connection_string, **self._client_kwargs)
-            _ServiceTest.async_service_client = AsyncBlobServiceClient.from_connection_string(conn_str=connection_string, **self._client_kwargs)
+            if self.args.use_entra_id:
+                tenant_id = self.get_from_env("AZURE-STORAGE-BLOB_TENANT_ID")
+                client_id = self.get_from_env("AZURE-STORAGE-BLOB_CLIENT_ID")
+                client_secret = self.get_from_env("AZURE-STORAGE-BLOB_CLIENT_SECRET")
+                sync_token_credential = SyncClientSecretCredential(
+                    tenant_id,
+                    client_id,
+                    client_secret
+                )
+                async_token_credential = AsyncClientSecretCredential(
+                    tenant_id,
+                    client_id,
+                    client_secret
+                )
+                account_name = self.get_from_env("AZURE_STORAGE_ACCOUNT_NAME")
+                url = f"https://{account_name}.blob.core.windows.net"
```

Comment:
Maybe just a comment here for reference noting that this will only work on the public cloud (though probably not likely anyone would be running this on the China or Government clouds...)
```suggestion
                # We assume these tests will only be run on the Azure public cloud for now.
                url = f"https://{account_name}.blob.core.windows.net"
```